app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>If you're thinking about changing careers to become a teacher, knowing where to start and understanding what your options are can seem overwhelming. There's lots of dedicated support available to help you navigate the change.</p>
+    <h2 class="heading--box-blue">Get advice and guidance from an expert</h2>
+    <h3>Teacher training advisers</h3>
+    <p>Our experienced <a href="/teacher-training-advisers">teacher training advisers</a> offer free one-to-one support to thousands of people each year to support them through a career change into teaching. All of our advisers have years of teaching experience, so can also give practical advise and insights on what life in the classroom is like.</p>
+    <p>If you have reservations about becoming a teacher, they may know of support available that you're not aware of, or could give practical advice on how other career changers have tackled barriers to becoming a teacher.</p>
+    <p>You can chat to your dedicated adviser by phone, text or email.</p>
```

Comment:
```suggestion
    <p>You can chat to your dedicated advisers by phone, text or email.</p>
```

Reply:
Do people get multiple advisers or just one?

Reply:
Adviser should be singular? You are only matched with one!

Reply:
Ignore me, I read it as our dedicated advisers!